eng/ci/templates/official/jobs/run-benchmarks.yml

```diff
@@ -55,6 +58,9 @@ jobs:
   - script: dotnet tool install -g Microsoft.Crank.Agent --version "0.2.0-*"
     displayName: Install Microsoft.Crank.Agent tool
 
+  - script: dotnet tool install -g Microsoft.Crank.Controller --version "0.2.0-*"
+    displayName: Install Microsoft.Crank.Controller
```

Comment:
small nit: you end the previous step with `tool` but not this one. It's completely irrelevant in the end, just a slight eye twitch from me that's all.

Reply:
Fixed to be consistent with other displayName.